JsonSchema/SchemaKeywordRegistry.cs

```diff
@@ -21,78 +22,82 @@ namespace Json.Schema;
 public static class SchemaKeywordRegistry
 {
 	private static readonly ConcurrentDictionary<string, Type> _keywords;
-	private static readonly ConcurrentDictionary<Type, JsonTypeInfo> _keywordTypeInfos;
+	private static readonly ConcurrentDictionary<Type, JsonSerializerContext> _keywordTypeInfoResolvers = new();
```

Comment:
It doesn't look like this is ever initialized with the default keyword set.

Reply:
It isn't, those are from the default JsonSchema serializer context. I can add a comment.

Reply:
Okay... this is importing new contexts.  Got it.